cpu/esp32/esp-ble-nimble/doc.txt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2022 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        cpu_esp32_esp_ble_nimble ESP32 Bluetooth LE HCI for NimBLE host
+ * @ingroup         cpu_esp32
+ * @brief           ESP32 Bluetooth LE HCI implementaion for NimBLE host
```

Comment:
```suggestion
 * @brief           ESP32 Bluetooth LE HCI implementation for NimBLE host
```

Reply:
Yes, I haven't gone through the static test output yet :wink: